src/tomczyk-kamil/Fsd.Kamil.Cs/Fsd.Kamil.Cs.Ex4/Fsd.Kamil.Cs.Ex4.Web/Controllers/ProductController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Fsd.Kamil.Cs.Ex4.Data.Entities;
+using Fsd.Kamil.Cs.Ex4.Models.Products;
+using Fsd.Kamil.Cs.Ex4.Services.Services.Products;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+
+namespace Fsd.Kamil.Cs.Ex4.Web.Controllers
+{
+    public class ProductController : Controller
+    {
+        private IManualProductService _productService;
+
+        public ProductController()
+        {
+            _productService = new ManualProductService();
+        }
+
+        public ActionResult Index()
+        {
+            return View();
+        }
+        public ActionResult Product()
+        {
+            string page = Request.QueryString["page"];
+            string items = Request.QueryString["items"];
+
+            IEnumerable<Product> products = _productService.GetProducts();
+
+            if (page == null || items == null)
```

Comment:
ten if mi sie nie podoba - powinna być 1 metoda która dostaje te wartości lub jak ich nie ma to wartości domyślne